### PR TITLE
feat(frontend): register merchant API implementation

### PR DESCRIPTION
**Pull request by markjoshuacarmen**
## Ticket/s

- [Merchant Registration API](https://trello.com/c/aYME5Zxo/151-merchant-registration)

## Changes

- Register Merchant API Implementation

## Screenshot

image

## How to test?

- register to http://localhost:5173/register
- check db if new merchant is added

### CODE REVIEW

**Review by sandrex-SGV — CHANGES_REQUESTED** [review]
Validation does not work (see attached image).

image

**Review by sandrex-SGV — APPROVED** [review]
lgtm

File: apps/api/src/app.module.ts [hunk 1]
```diff
@@ -18,9 +18,7 @@ import { RatingModule } from './rating/rating.module';
   imports: [
     //use for local db test: mongodb://localhost:27017/test
     //use for central db test: mongodb+srv://user:aZkh7DhYjrP2i52U@personalproject.fzwyrwp.mongodb.net/POC-EYCommerce
-    MongooseModule.forRoot(
-      'mongodb+srv://user:aZkh7DhYjrP2i52U@personalproject.fzwyrwp.mongodb.net/POC-EYCommerce',
-    ),
+    MongooseModule.forRoot('mongodb://localhost:27017/test'),
```
Comment (TreshAnn) [inline apps/api/src/app.module.ts]: make sure to use the connection string for central db before merging.

**Review by TreshAnn — APPROVED** [review]
LGTM

File: apps/api/src/app.module.ts (hunk 1 above)
Comment (markjoshuacarmen) [reply apps/api/src/app.module.ts]: this is noted thanks

**Pull request merged**